src/p1_mysql_auth.erl

```diff
@@ -72,8 +73,27 @@ do_old_auth(Sock, RecvPid, SeqNum, User, Password, Salt1, LogFun) ->
 %% Returns : result of p1_mysql_conn:do_recv/3
 %%--------------------------------------------------------------------
 do_new_auth(Sock, RecvPid, SeqNum, User, Password, Salt1, Salt2, LogFun) ->
+    do_new_auth(Sock, RecvPid, SeqNum, User, Password, Salt1, Salt2, undefinded, LogFun)
```

Comment:
That `undefinded` value here is suspicious.

Reply:
Is it still hardcoded value MaxPacketSize (~1mb)?